### PR TITLE
Show active environment on startup | closes #12

### LINKED ISSUE #12: Initially setting the venv to the terminal's venv on startup
Hello!

Could it be possible to initially set the venv to the one which is already activated on the terminal initially on nvim's startup (i.e., when calling swenv's `setup()`)?

Thanks!

### PR DESCRIPTION
**Pull request by skolj**

### CODE REVIEW

File: lua/swenv/api.lua [hunk 1]
```diff
@@ -28,6 +28,55 @@ local set_venv = function(venv)
   end
 end
 
+---
+---Checks who appears first in PATH. Returns `true` if `first` appears first and `false` otherwise
+---
+---@param first string|nil
+---@param second string|nil
+---@return boolean
+local hasHighPriorityInPath = function (first, second)
```
Comment (AckslD) [inline lua/swenv/api.lua]: [nit] since everything else is snake_case could this also be?

File: lua/swenv/api.lua [hunk 2]
```diff
@@ -28,6 +28,55 @@ local set_venv = function(venv)
...
+  return string.find(ORIGINAL_PATH, first) < string.find(ORIGINAL_PATH, second)
+end
+
+M.init = function()
+  local success, Path = pcall(require, 'plenary.path')
+  if not success then
+    vim.notify('Could not require plenary: ' .. Path, vim.log.levels.WARN)
+    return
+  end
+  local venv = nil
```
Comment (AckslD) [inline lua/swenv/api.lua]: [nit]

```suggestion
  local venv
```

File: lua/swenv/init.lua [hunk 3]
```diff
@@ -8,6 +8,7 @@ end
 
 M.setup = function(opts)
   update_config(opts)
+  require('swenv.api').init()
```
Comment (AckslD) [inline lua/swenv/init.lua:11]: [question] should this be optional? previously the `api` module was not loaded on startup with `plenary`. not sure if it's important though

**Review by AckslD — COMMENTED** [review]
Thanks @skolj, this looks great! Just some minor comments/questions

File: lua/swenv/api.lua (hunk 1 above)
Comment (skolj) [reply lua/swenv/api.lua]: sure, force of habit I guess.

File: lua/swenv/init.lua (hunk 3 above)
Comment (skolj) [reply lua/swenv/init.lua:11]: umm sure, how about invoking it only when the filetype is python?

Comment (skolj) [reply lua/swenv/init.lua:11]: umm I'm not quite sure I understand what you mean. But I notice that we could invoke it iff the filetype is python (or not, we could always open a .py, however, we could only show the statusline entry iff the filetype is python). what do you think?

Comment (AckslD) [reply lua/swenv/init.lua:11]: Yeah nevermind actually :) The user can just choose to call setup later if they want :+1:

**Pull request merged**